hpOneView/resources/servers/id_pools_vwwn_ranges.py

```diff
@@ -0,0 +1,205 @@
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+
+standard_library.install_aliases()
+
+__title__ = 'id-pools-wwwn-ranges'
+__version__ = '0.0.1'
+__copyright__ = '(C) Copyright (2012-2016) Hewlett Packard Enterprise Development LP'
+__license__ = 'MIT'
+__status__ = 'Development'
+
+from hpOneView.resources.resource import ResourceClient
+
+
+class IdPoolsVwwnRanges(object):
+    URI = '/rest/id-pools/vwwn/ranges'
+
+    def __init__(self, con):
+        self._connection = con
+        self._client = ResourceClient(con, self.URI)
+
+    def create(self, resource, timeout=-1):
+        """
+        Creates a vWWN range.
+
+        A range can be one of two types based upon the range category specified: Generated or Custom. The Generated
+        range type automatically assigns start and end addresses to the range. The Custom range type requires a start
+        address to be specified. The end address may also be specified but is optional.
+
+        Args:
+            resource (dict): Object to create
+            timeout:
+                Timeout in seconds. Wait task completion by default. The timeout does not abort the operation
+                in OneView, just stops waiting for its completion.
+
+        Returns:
+            dict: Created id range
+        """
+        return self._client.create(resource, timeout=timeout)
+
+    def get(self, id_or_uri):
+        """
+        Gets a vWWN range.
+
+        Using the allocator and collector associated with the range, IDs may be allocated from or collected back to the
+        range.
+
+        Args:
+            id_or_uri: Could be either the vWWN range id or uri
+
+        Returns:
+            dict: vWWN range
+        """
+        return self._client.get(id_or_uri)
+
+    def enable(self, information, id_or_uri=None, timeout=-1):
+        """
```

Comment:
To 'keep it simple' we are choosing pass a dictionary similar to json described on the OneView documentation(not a boolean).